website/components/Modals/ShareModal.tsx

```diff
@@ -51,7 +51,10 @@ const ShareModal: React.FC<ShareModalProps> = (props: ShareModalProps) => {
   }
 
   const params: URLSearchParams = new URLSearchParams(serializedViewerParams);
-  const shareUrl = params.size > 0 ? `${baseUrl}?${params.toString()}` : baseUrl;
+  let shareUrl = params.size > 0 ? `${baseUrl}?${params.toString()}` : baseUrl;
+
+  // Decode specifically colons and commas for better readability + decreased char count
+  shareUrl = shareUrl.replace(ENCODED_COLON_REGEX, ":").replace(ENCODED_COMMA_REGEX, ",");
```

Comment:
Fix this so it only escapes `params.toString()` and not the base Url.

Reply:
make sure its only replacing on the part of the params excluding the serializedUrl (I believe the `url=` param should stay untouched)